book/src/pong-tutorial/pong-tutorial-02.md

```diff
@@ -28,14 +28,14 @@ Then, in `main.rs` declare a module:
 mod pong;
 ```
 
-And in the `run()` function add:
+And in main.rs, before `fn main()`, add this import:
```

Comment:
I would rephrase this as  `And in main.rs at the top of the file add this import.`